server/mediapublic/mediapublic/models.py

```diff
@@ -259,8 +261,59 @@ def to_dict(self):
             secondary_website = self.secondary_website,
             creation_datetime = str(self.creation_datetime),
         )
-        return resp    
-    
+        return resp
+
+class PlaylistAssignments(Base, CreationMixin):
+    __tablename__ = 'playlist_assignments'
+    id = Column(Integer, primary_key=True)
+    playlist_id = Column(Integer, ForeignKey('playlists.id'))
+    recording_id = Column(Integer, ForeignKey('recordings.id'))
```

Comment:
We'll get playlist_id from the request.matchdict['pid'], so that isn't required, but to use check_payload, we'll need recording_id to be a ReqColumn class, not Column.